hibernate-core/src/main/java/org/hibernate/resource/transaction/TransactionCoordinator.java

```diff
@@ -113,6 +114,8 @@
 
 	public int getTimeOut();
 
+	public void setExceptionMapper(ExceptionMapper exceptionMapper);
+
```

Comment:
I'm not overly a fan of setting the `ExceptionMapper`.  I'd much rather see this handled via something like a new `org.hibernate.engine.spi.SessionOwner` method, or another means of injecting this into the Session itself.